versions/run.hercules.ver

```diff
@@ -0,0 +1,10 @@
+export stack_intel_ver=2021.9.0
+export stack_impi_ver=2021.9.0
+
+export ncl_ver=6.6.2
+
+source "${HOMEgfs:-}/versions/run.spack.ver"
+
+# wgrib2 and cdo are different on Hercules from all the other systems
+export wgrib2_ver=3.1.1
+export cdo_ver=2.2.0
```

Comment:
Is it possible to update the versions for the other machines so they match again?

Reply:
Unfortunately not.  These versions are not available on the other machines under spack-stack.  It looks like the standard versions (`cdo/2.0.5` and `wgrib2/2.0.8`) [don't compile on Hercules](https://github.com/JCSDA/spack-stack/blob/8780bcdff613e970e2cf881301587b72bd7f3ef8/configs/sites/hercules/packages.yaml#L24-L30).  There are plans to upgrade to these versions for all systems in future releases (https://github.com/JCSDA/spack-stack/pull/884 and https://github.com/JCSDA/spack-stack/issues/859).